### PR TITLE
Updated to latest issue and PR templates

### PR DESCRIPTION
**Pull request by ricellis**
Thanks for your hard work, please ensure all items are complete before opening.

- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](https://github.com/cloudant/java-cloudant/blob/master/DCO1.1.txt)
- [x] You have added tests for any code changes - N/A
- [x] You have updated the [CHANGES.md](https://github.com/cloudant/java-cloudant/blob/master/CHANGES.md) - N/A
- [x] You have completed the PR template below:

## What

Updated issue and PR templates and contributing file.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,35 @@
...
+<!--
+Got a 400 bad request
+-->
+
+## Environment details
+<!--
+- Version(s) that are affected by this issue.
+    > 2.0.2
+- Java version including vendor and platform
+    >1.8.0_151 OpenJDK macOS
```
Comment (emlaver) [inline .github/ISSUE_TEMPLATE.md]: Add space

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,6 +1,35 @@
...
+- Using the optional okhttp dependency.
+    >Yes|No
```
Comment (emlaver) [inline .github/ISSUE_TEMPLATE.md]: Add space

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (ricellis) [reply .github/ISSUE_TEMPLATE.md]: Fixed in ac0cfcbc5fd6e3c8cc571f80c5faeb975941bcaf

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (ricellis) [reply .github/ISSUE_TEMPLATE.md]: Fixed in ac0cfcbc5fd6e3c8cc571f80c5faeb975941bcaf

**Pull request merged**